sam_stonefish_sim/data/sam_auv/sam_auv.xml

```diff
@@ -256,6 +256,16 @@
         <ros_publisher topic="/$(arg robot_name)/perception/mbes"/>
     </sensor>
 
+    <sensor name="side_scan_sonar" type="sss" rate="5.0">
+        <link name="Vehicle"/>
+        <origin rpy="0.0 3.1416 1.5707" xyz="0.7 0.0 -0.07"/>
+        <specs bins="500" lines="400" horizontal_beam_width="0.5" vertical_beam_width="60.0" vertical_tilt="60.0"/>
+        <settings range_min="1.0" range_max="100.0" gain="1.0"/>
+        <noise multiplicative="0.02" additive="0.04"/>
+        <display colormap="hot"/>
+        <ros_publisher topic="/$(arg robot_name)/perception/sss"/>
```

Comment:
Let's call this `/$(arg robot_name)/sim/sidescan` instead, then we'll add a conversion to the actual sidescan topic

Reply:
Which should be named `/$(arg robot_name)/payload/sidescan` according to [the spec](https://github.com/smarc-project/smarc_msgs#payloads)